### ISSUE TITLE
Bottom Stats -> Albums

### ISSUE THREAD

**Issue by ingbife** [gjr-osweb/libratus#21]
Hi mindhaq

the output of LATEST ALBUMS (ID), LATEST ALBUMS (MTIME) and LATEST ALBUMS (DATE) is the same ?

Please see an example here:
![output-the-same](https://cloud.githubusercontent.com/assets/12975007/8266156/9acedcc6-171d-11e5-995c-b74e199075e6.PNG)
![output-the-same db](https://cloud.githubusercontent.com/assets/12975007/8266163/142be88e-171e-11e5-9e17-94671ef41001.PNG)

**Comment by acrylian** [comment]
This is not theme related actually. Some explaination how Zenphoto uses these orders: 
- ID => discovery order from the file system (Zenphoto's gallery is file system based), not necessarily the same as the date
- Date => Date uploaded, not necessarily matchin the id but mostly (and it can be changed)
- mtime => Is the file modification date so folders actually don't have that. Albums use the date of the latest image for the updated date (off hand spoken).
- updateddate => The date you last added something to an album (title etc changes don't count)
- publishdate => is actually the date used for scheduled (un)publishing 

In short, yes it might happen that these return the same order.
